recipes/icu/bld.bat

```diff
@@ -0,0 +1,15 @@
+if "%ARCH%"=="32" (
```

Comment:
Alright, let's leave it alone then. Thanks for checking though. This will be helpful information when it comes back up again. :smile: